doc/level1.rst

```diff
@@ -1082,6 +1083,48 @@ Earlier choices are better than later ones.
    background. Set your ``accounts[addr].prefer_encrypt`` to
    ``nopreference`` and start sending Autocrypt headers.
 
+.. _enabling-disabling:
+
+Disabling Autocrypt
++++++++++++++++++++
+
+Once Autocrypt is enabled for a given account
+(``accounts[addr].enabled`` is set to ``true``), the user might choose
+to disable it.  By default, disabling should only set
+``accounts[addr].enabled`` to ``false``, and it SHOULD NOT destroy
+``accounts[addr].secret_key``.  This preserves the user's ability to
+read old encrypted e-mails, as well as being able to read encrypted
+e-mails that arrive after the user has disabled Autocrypt.
+
+The act of re-enabling Autocrypt after it was disabled SHOULD leave
+``accounts[addr].secret_key`` and ``accounts[addr].public_key``
+intact, so that the user continues using the same key.
+
+Destroying Secret Key Material
+++++++++++++++++++++++++++++++
+
+When disabling Autocrypt for an account, a Level 1 MUA MAY offer the
+user an opportunity to also destroy the secret key material for that
+account.  Since Autocrypt clients generally do not discuss secret keys
+with users, a MUA offering this choice should use a phrase like
+"destroy access to all encrypted messages", rather than referring to
+"keys" or "key material".
+
+A MUA that allows the user this opportunity SHOULD clearly indicate to
+the user that the destruction of this secret key material will leave
+them unable to read their previously-received encrypted messages as
```

Comment:
whether or not past messages can be read depends on how a MUA handles storage of incoming messages. Delta keeps decrypted messages in its local store whereas e.g. k9/thunderbird use IMAP and thus pgp-encrypted messages for their message store.  

i guess it might be better overall to talk unreadable messages in user-facing warnings aka something like "future received message may be unreadable if you destroy your secrets" .

Reply:
Sure, a MUA that does *not* destroy access to the messages should not indicate that they will be destroyed if they are not destroyed.  I'll push a new revision that takes this into account.